CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ For migration information, you can always have a look at https://liip-drifter.re
 
 ## [Unreleased]
 
+### Added
+
+- Add parameters memory & cpus and changed Vagrantfile and Vagrantfile.dist accordingly
```

Comment:
Can you please change the message to something more meaningful to the end user? For example "Allow CPU count and memory to be configured in parameters.yml"